cachito/web/content_manifest.py

```diff
@@ -12,7 +12,6 @@
 from cachito.web.utils import deep_sort_icm
 from cachito.workers.pkg_managers import gomod
 
-
```

Comment:
Can you move this change to https://github.com/release-engineering/cachito/pull/564/commits/b491af65ae02c577309ef13a7e497f1dccd2289a ?